week0003/hoangdangduy/java/Process.java

```diff
@@ -0,0 +1,84 @@
+import java.util.HashMap;
+
+public class Process {
+	private static final String add = "A";
+	private static final String deposite = "D";
+	private static final String withdraw = "W";
+	
+	public HashMap<String, Double> process(String data, HashMap<String, Double> listAccount) {
+		Account account = new Account();
+		String[] inforTransaction = data.split(" ");
+		
+		switch (inforTransaction[0]) {
+		case add:
+			account = createAccount(inforTransaction[1], Double.parseDouble(inforTransaction[2]), listAccount);
+			return editListAccAfterAction(account, listAccount);
+		case deposite:
+			account = depositMoney(inforTransaction[1], Double.parseDouble(inforTransaction[2]), listAccount);
+			return editListAccAfterAction(account, listAccount);
+		case withdraw:
+			account = withdrawMoney(inforTransaction[1], Double.parseDouble(inforTransaction[2]), listAccount);
+			return editListAccAfterAction(account, listAccount);
+		default:
+			Response response = new Response();
+			response.showMessage("Invalid");
+			break;
+		}
+		return listAccount;
+	}
+	
+	public Account createAccount(String name, Double money, HashMap<String, Double> listAccount){
```

Comment:
Remember: The name is case-insensitive. So you can transform it to lowercase or uppercase before creating a new account.